lib/puma/server.rb

```diff
@@ -111,16 +113,18 @@ def inherit_binder(bind)
       def cork_socket(socket)
         begin
           socket.setsockopt(6, 3, 1) if socket.kind_of? TCPSocket
-        rescue IOError, SystemCallError
+        rescue IOError, SystemCallError => e
           Thread.current.purge_interrupt_queue if Thread.current.respond_to? :purge_interrupt_queue
+          @debug_logger.error_dump(e)
```

Comment:
This rescue and the one in `uncork` aren't really worth logging, I think.

Reply:
Done